src/plugins/kibana/public/management/app.html

```diff
@@ -10,9 +10,11 @@
       <li
         ng-if="sectionName"
         ng-repeat="item in section.items.inOrder"
-        ng-class="item.class">
+        ng-class="item.class"
+        tooltip="{{::item.tooltip}}" tooltip-placement="bottom" tooltip-popup-delay="400" tooltip-append-to-body="1"
+      >
 
-        <a class="navbar-link" kbn-href="{{::item.url}}" data-test-subj="{{::item.name}}">
+        <a class="navbar-link" ng-class="{ 'is-navbar-nav-link-disabled': !item.path }" kbn-href="{{::item.url}}" data-test-subj="{{::item.name}}">
```

Comment:
Ah, yeah. The change you mentioned [here](https://github.com/elastic/kibana/pull/7573/files/1fd7292183804ed2d8e87a673b6922fb84115775#r68869278) should resolve that.